packages/tpetra/core/example/Finite-Element-Assembly/README.md

```diff
@@ -15,8 +16,17 @@ Files
   - Provides helpers for processing command line options.
 - `fem_assembly_main`
   - Contains the main() function for the application.
+
+### Finite Element Mesh Assembly Example Sources
```

Comment:
Wait, where did these changes come from?

Reply:
@mhoemmen When I removed the DynamicProfile versions, I removed too much of the actual explanations of the options from the README, so today I put them back.

Reply:
Thanks @brian-kelley ! :-)